docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -188,33 +196,222 @@ if (sdlManager.getSystemCapabilityManager().isCapabilitySupported(SystemCapabili
+sdlManager.send(request: setData, responseHandler: { (request, response, error) in
+    guard response?.success.boolValue == true else { return }
+    <#Seat Data Updated#>
+})
+```
+!@
+
+### Getting Remote Control Modules
+When connected to Core v6.0+ you will want to save the moduleID for future use. The `moduleID` is contained within `moduleInfo` in the remote control capability module on Core v6.0+. If Core is less then 6.0 you shoud still get remote control module data but you do not need to save the `moduleID`.
+
+@![iOS]
+##### Objective-C
+```objc
+[self.sdlManager.systemCapabilityManager subscribeToCapabilityType:SDLSystemCapabilityTypeRemoteControl withBlock:^(SDLSystemCapability * _Nonnull capability) {
+    if(!capability.remoteControlCapability) { return; }
+    <#Save Remote Capabilities#>
+}];
+```
+##### Swift
+```swift
+sdlManager.systemCapabilityManager.subscribe(toCapabilityType: .remoteControl, with: { capability in
+    guard (capability.remoteControlCapability != nil) else { return }
+    <#Save Remote Capabilities#>
+})
+
+```
+!@
+
+With the saved remote control capabilities struct you can build a UI to display modules to the user.
+
+!!! Note
+`ModuleID` is only required for Core v6.0+. Examples will be shown later in the documentation on how to use `moduleID`.
+!!!
+
+@![iOS]
+##### Objective-C
+```objc
+NSString *selectedModuleID = <#SelectedModule#>.moduleInfo.moduleID;
+```
+
+##### Swift
+```swift
+let selectedModuleID = <#SelectedModule#>.moduleInfo?.moduleId
+```
+!@
+
+### Getting Consent to Control a Module
+This feature is only available for Core v6.0+. Some OEMs may wish to ask the driver for consent before a user can control a module. The `SDLGetInteriorVehicleDataConsent` RPC will alert the driver for consent in some OEM head units if the module if not free (another user has control) and `allowMultipleAccess` (multiple users can access/set the data at the same time) is `true`. `allowMultipleAccess` is part of the `moduleInfo` in the module object.
+
+Check the `allowed` property in the `SDLGetInteriorVehicleDataConsentResponse` to see what modules can be controlled. Note the order of the `allowed` array is 1-1 with the `moduleIDs` array you passed into the `SDLGetInteriorVehicleDataConsent` RPC.
+
+!!! Note
+You should always try to get consent before setting any module data. If consent is not granted you should not attempt to set any module's data.
+!!!
+
+@![iOS]
+##### Objective-C
+```objc
+SDLGetInteriorVehicleDataConsent *getInteriorVehicleDataConsent = [[SDLGetInteriorVehicleDataConsent alloc] initWithModuleType:@"<#ModuleType#>" moduleIds:@[@"<#ModuleID#>",@"<#ModuleID#>",@"<#ModuleID#>"]];
+[self.sdlManager sendRequest:getInteriorVehicleDataConsent withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+    if (!response.success) { return; }
+    <#Allowed is an arry of true or false values#>
```

Comment:
```suggestion
    <#Allowed is an array of true or false values#>
```

Reply:
You're not converting the response to the `SDLGetInteriorVehicleDataConsentResponse` or getting the allowed array.